src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -20,23 +20,17 @@ import { RoomModalOngoingEvent, ScheduleItem } from "..";
 
 import "./RoomModal.scss";
 
-interface RoomModalProps {
+type RoomModalProps = {
```

Comment:
```suggestion
export type RoomModalProps = {
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1129/commits/53f57c40e24983470b7552d4ebed6f1715ff74ba